events/app/controllers.js

```diff
@@ -6,7 +6,9 @@ export async function renderEvent(ctx, next) {
   const id = `${ctx.params.id}`;
   const format = ctx.request.query.format;
   const isPreview = Boolean(ctx.params.preview);
-  const event = await prismic.getEvent(id, isPreview ? ctx.request : null);
+  const eventPromise = prismic.getEvent(id, isPreview ? ctx.request : null);
+  const globalAlertPromise = prismic.getGlobalAlert();
```

Comment:
Let's shift this into `interval-cache`.

Reply:
Yep, that's the right way to do it – thought I was overlooking something